.github/workflows/deploy.yml

```diff
@@ -0,0 +1,88 @@
+# Configuration for publishing archives to PyPI and documentation pages to
+# GitHub Pages using GitHub Actions
+name: deploy
+
+# Only run for pushes to the master branch and releases.
+on:
+  push:
+    branches:
+      - master
+      # FOR TESTING ONLY! REMOVE BEFORE MERGING
+      - deploy-pypi
+  release:
+    types:
+      - published
+
+jobs:
+
+  #############################################################################
+  # Publish built wheels and source archives to PyPI and test PyPI
+  pypi:
+    name: PyPI
+    runs-on: ubuntu-latest
+    # Only publish from the origin repository, not forks
+    if: github.repository == 'fatiando/pooch'
+
+    steps:
+
+      # Checks-out your repository under $GITHUB_WORKSPACE
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Need to fetch more than the last commit so that versioneer can
+          # create the correct version string. If the number of commits since
+          # the last release is greater than this, the version still be wrong.
+          # Increase if necessary.
+          fetch-depth: 100
+
+      # Need the tags so that versioneer can form a valid version number
+      - name: Fetch git tags
+        run: git fetch origin 'refs/tags/*:refs/tags/*'
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Install requirements
+        run: python -m pip install setuptools twine wheel
+
+      - name: List installed packages
+        run: python -m pip freeze
+
+      - name: Build source and wheel distributions
+        run: |
+          # Change the versioneer format to "pre" so that the commit hash isn't
+          # included (PyPI doesn't allow it). Can't do this permanently because
+          # we rely the hash to indicate to the tests that this is a local
+          # verison instead of a published version.
+          sed --in-place "s/pep440/pep440-pre/g" setup.cfg
+          python setup.py sdist bdist_wheel
+          echo ""
+          echo "Generated files:"
+          ls -lh dist/
+
+      - name: Check the archives
+        run: twine check dist/*
+
+      - name: Publish to Test PyPI
+        if: success()
```

Comment:
I think `if: success()` is the default.

Reply:
👍🏽 I thought so but just wanted to be safe